### ISSUE TITLE
AEAudioFilePlayer playAtTime: is not frame-accurate

### ISSUE THREAD

**Issue by jeremywhuff** [TheAmazingAudioEngine/TheAmazingAudioEngine#224]
TLDR - playAtTime is only accurate to the length of a render cycle, not actual host ticks.

---

I am trying to use the currentTime property in conjunction with the playAtTime: in order to make an AEAudioFilePlayer start playing at a moment in the future, perfectly synchronized with another AEAudioFilePlayer. Basically I'm saying, "Start playing at this exact moment in the future, from this exact frame in the audio file."

Say I set this:
myPlayer.currentTime = x;
[myPlayer playAtTime:y];

Then I find that myPlayer is 2ms too late. So I make an adjustment:
myPlayer.currentTime = x;
[myPlayer playAtTime:y - 2ms_in_ticks];

I would expect that myPlayer will play 2ms earlier, but it's still 2ms too late. To experiment, I try this:
myPlayer.currentTime = x;
[myPlayer playAtTime:y - 4ms_in_ticks];

Now I find that myPlayer is playing 8.4ms too early. So it appears that playAtTime is quantizing the actual start time by 11.6ms increments.

My audio render cycles are 512 frames long, and sample rate is 44100Hz. So this quantization seems to be equal to the length of a render cycle (512 / 44100 = 0.0116).
